### ISSUE TITLE
Default overwrites

### ISSUE THREAD

**Issue by tomy8910** [typicode/lowdb#244]
This is more of a question with the API. Will writing defaults overwrite the current data.

{
  “name”: “This is not a default name”
}

If I set db.defaults({name:”default name”}), will the current “This is not a default name” be replaced?

**Comment by smartclash** [comment]
Nope. It will not @tomy8910. You can now safely close this issue.
